.gitignore

```diff
@@ -4,6 +4,7 @@ zuliprc
 cov_html/*
 .coverage
 .mypy_cache
+.mypy-cache
```

Comment:
The cache file created was `.mypy-cache` and not `mypy_cache` due to which i added it to gitignore.

Reply:
True. I'll have a look.